edgelet/iotedged/src/settings.rs

```diff
@@ -353,6 +363,24 @@ mod tests {
             .expect("certificates not configured");
     }
 
+    #[test]
+    fn dps_prov_symmetric_key_get_settings() {
+        let settings = Settings::<DockerConfig>::new(Some(GOOD_SETTINGS_DPS_SYM_KEY));
+        println!("{:?}", settings);
+        assert!(settings.is_ok());
+        let s = settings.unwrap();
+        match s.provisioning() {
+            Provisioning::Dps(ref dps) => {
+                assert_eq!(dps.global_endpoint().scheme(), "scheme");
+                assert_eq!(dps.global_endpoint().host_str().unwrap(), "jibba-jabba.net");
+                assert_eq!(dps.scope_id(), "i got no time for the jibba-jabba");
+                assert_eq!(dps.registration_id(), "register me foo");
```

Comment:
I'm pretty certain Mr. T would say "register me, fool" as in "I pity the fool".

Reply:
Thanks I will fix the typo :). It is important that Mr. T is well represented.